package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "pnpm -r run dev",
     "build": "pnpm -r run build",
-    "publish:playgrounds": "pnpm pkg-pr-new publish ./playgrounds/* ./packages/cli --template=\"./templates/*\"",
+    "publish:playgrounds": "pnpm pkg-pr-new publish './playgrounds/*' './packages/cli' --template './templates/*'",
```

Comment:
let's remove '' from one of either playgrounds or cli so when publishing next times, we'd see if without ' it'd fail! So we do not fail for the current users

Reply:
Done! From what I've learnt it shouldn't make a difference, it's just "more correct" to do it this way.